lib/runner.js

```diff
@@ -74,7 +75,8 @@ function run (cliOptionsOrConfig, done) {
     method: 'POST',
     headers: {
       'Content-Type': 'application/json'
-    }
+    },
+    lookup: (hostname, options, callback) => dns.lookup(hostname, { ...options, verbatim: false }, callback)
```

Comment:
Ditto. Or even better, how about implementing the overriden version of `dns.lookup` in a single file and importing that from there?

Reply:
Good idea! Done.